tests/StackExchange.Redis.Tests/BasicOps.cs

```diff
@@ -431,6 +431,22 @@ public void ShouldUseSharedMuxer()
             }
         }
 
+        [Fact]
+        public async Task Copy()
+        {
+            using (var muxer = Create())
+            {
+                var db = muxer.GetDatabase();
+                var src = Me();
+                var dest = Me() + "2";
+                _ = db.StringSetAsync(src, "Heyyyyy");
```

Comment:
we should probably also have:

- a test of the cross-DB functionality
- a test of what scenario would cause it to return `false` - presumably missing source?

but other than that: looking great, thanks! Note there also a releasenotes.md - could you add to that too, following the pattern?

Reply:
Maybe someone can **help**: Having trouble with copying to another database - I don't know why but the test skips (at least not failing)

Also there is one test that returns `false` when the destination key exists and we're not using REPLACE (just added now).

Reply:
Where is the releasenotes.md? Can't find it

Reply:
Fixed current tests/commands - will try and add a few more tests today!

Reply:
Thank you! @NickCraver